Library/ViewModels/SearchViewModelTests.swift

```diff
@@ -508,8 +508,8 @@ internal final class SearchViewModelTests: TestCase {
       )
       projects.assertLastValue(popularProjects, "Brings back popular projects immediately.")
 
-      XCTAssertEqual(["Page Viewed"], self.dataLakeTrackingClient.events)
-      XCTAssertEqual(["Page Viewed"], self.segmentTrackingClient.events)
+      XCTAssertEqual(["Page Viewed", "Page Viewed"], self.dataLakeTrackingClient.events)
```

Comment:
Why is there an additional Page Viewed event here now?

Reply:
This has been fixed